src/infrastructure/AccountHttp.ts

```diff
@@ -67,7 +60,33 @@ export class AccountHttp extends Http implements AccountRepository {
      * @returns Observable<AccountInfo>
      */
     public getAccountInfo(address: Address): Observable<AccountInfo> {
-        return observableFrom(this.accountRoutesApi.getAccountInfo(address.plain())).pipe(map((accountInfoDTO) => {
+
```

Comment:
:warning: not correct.

you should be copying over the `nem2-library/src/api/*` classes into the SDK because they are **swagger generated**.